app/radiosonde_auto_rx_listener.py

```diff
@@ -21,10 +21,97 @@ def __init__(self):
 
         logger.info("AsyncRadiosondeAutoRxListener initialized.")
 
-
     async def start(self):
+        logger.info("Starting AsyncRadiosondeAutoRxListener...")
+        if self._settings.fetch_data:
+            logger.info("Fetching data from online sites.")
+            await self._listen_web()
+        else:
+            logger.info("Fetching data from local udp.")
+            await self._listen_udp()
+
+    async def _listen_web(self):
+        """Uses async requests to fetch data from online sources."""
+        url = f"https://s1.radiosondy.info/export/export_map.php?live_map=1&_={int(datetime.now().timestamp() * 1000)}"
+
+        headers = {
+            "Accept": "application/json, text/javascript, */*; q=0.01",
+            "Accept-Language": "it-IT,it;q=0.9,en-US;q=0.8,en;q=0.7",
+            "Cache-Control": "no-cache",
+            "Connection": "keep-alive",
+            "Pragma": "no-cache",
+            "Referer": "https://s1.radiosondy.info/",
+            "Sec-Fetch-Dest": "empty",
+            "Sec-Fetch-Mode": "cors",
+            "Sec-Fetch-Site": "same-origin",
+            "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36",
+            "X-Requested-With": "XMLHttpRequest",
+            "sec-ch-ua": '"Google Chrome";v="131", "Chromium";v="131", "Not_A Brand";v="24"',
+            "sec-ch-ua-mobile": "?0",
+            "sec-ch-ua-platform": '"Windows"',
+        }
+        logger.info("Fetching data from online source now..")
+        while True:
+            logger.info(
+                "---------------------------------------\n   [READING DATA FROM ONLINE SOURCE]   \n---------------------------------------"
+            )
+            async with aiohttp.ClientSession() as session:
+                async with session.get(url, headers=headers) as response:
+                    if response.status != 200:
+                        logger.error(
+                            f"Failed to fetch data from online source. Status code: {response.status}"
+                        )
+                        await asyncio.sleep(60)
+                        continue
+                    data = await response.json()
+                    if "features" in data:
+                        for feature in data["features"]:
+                            if (
+                                "geometry" in feature
+                                and feature["geometry"]["type"] == "Point"
+                            ):
+                                coords = (
```

Comment:
why not using 'latitude': '43.0685', 'longitude': '1.1515', on the root of the json?